setup.py

```diff
@@ -162,6 +183,9 @@ def run(self):
         "comtypes.tools",
         "comtypes.test",
     ],
+
+    platforms=["Windows"],
```

Comment:
This field is poorly documented, non-standardized, and ineffective. Let's avoid it.